app/pages/sections/order-items/catalogue-solutions/order-item/flat/patient/manifest.json

```diff
@@ -48,6 +48,12 @@
     }
   },
   "errorMessages": {
+    "PriceRequired": "Enter a price",
+    "PriceMustBeANumber": "Price must be a number",
+    "PriceMoreThan4dp": "Price must be to a maximum of 4 decimal places",
+    "PriceGreaterThanOrEqualToZero": "Price cannot be negative",
+    "PriceLessThanMax": "Maximum price exceeded, enter a lower price",
+    "PriceGreaterThanListPrice": "You’ve entered a price that is more than the list price, enter a lower price",
```

Comment:
It seems odd that these messages don't exist in this version. Are we not doing in validation on the price or are these messages define somewhere else?

Reply:
Ahh just a bad merge, I think my compare used the wrong lines.

Reply:
I think the same thing might have happened again – it looks we've still got the price error messages.

Reply:
I removed these properties from lower down in the object. I think it should be fine.

Reply:
And it should make further merges more seamless.

Reply:
If you have a look at the whole file you should be able to see that they're still there, above the quantity error messages.

Reply:
Oh, hang on a sec. Have they just been reordered for some reason? Let me check.